src/mixins/common-mixin.ts

```diff
@@ -40,11 +40,12 @@ export default class CommonMixin extends Vue {
     const validFlagArray = Object.keys(flags).map(key => flags[key])
 
     // Find the _first_ corresponding component that is invalid
-    const component = document.getElementById(components[validFlagArray.indexOf(false)])
+    const component = components[validFlagArray.indexOf(false)]
 
     // If there is an invalid component, scroll to it
     if (component) {
-      await this.scrollToTop(component)
+      const element = document.getElementById(component)
+      await this.scrollToTop(element)
```

Comment:
Currently Staff Change and alteration filings can file without having staff payment filled out, and with an invalid court order on the first time only. (Files but the schema doesn't allow the invalid filing to save)
Staff Special Resolutions can still be filed without the special resolution confirm checked (can file but schema doesn't allow saving) and staff payment checked.

Reply:
Staff payment issue: bcgov/entity#13981

Court order - no idea.

Do you think these ^^ are broken by this fix?

Reply:
Oh okay I see for staff payment. I don't think court order was broken by the fix, I think it was just exposed by it. Ultimately I think this solution works much better but doesn't totally fix it.

Reply:
Do you want to spent a bit of time qualifying these bugs (ie, did this fix break or expose them) and we fix all of them at the same time, or should we merge this and then work on those other 2 bugs next?

Reply:
I think that we should get this fix out probably sooner rather than later. The other bugs don't seen as important, what do you and @lewischenstudio think?

Reply:
I agree that filing without completing the Certify section is more serious than filing without valid Court Order or Staff Payment, since users will see the first but only staff will see the second (and we expect staff to be more careful).

Reply:
I agree, I'll dig into the court order issue more and try and figure it out.

Reply:
This solves the problem of scrolling but it probably doesn't fix the problem for the ticket. The ticket says submitting the filing is still enabled even though the certify component is incomplete/invalid. We can merge this PR first and fix the bug for the ticket after.

Reply:
Lewis, the scrolling and file-and-save block are handled in the same place: if an invalid component is found then the page scrolls and False (don't file) is returned. With the bug, the invalid component was not found and True (can file) was returned.

Reply:
In that case, this PR is the solution to solve the ticket 🙂

Reply:
Would it be beneficial to separate the scrolls to the incomplete filing and filing is valid? That would be fairly easy and would reduce the chance of making a filing with invalid sections?

Reply:
> I agree, I'll dig into the court order issue more and try and figure it out.

I'll create a ticket for the Court Order issue.